examples/src/main/scala/TraceExample.scala

```diff
@@ -133,17 +132,23 @@ object TraceExample extends IOApp.Simple {
             InstitutionServiceClient.apply[IO],
             UserDatabase.apply[IO]
           )
-          val resource: Resource[IO, Unit] =
-            Resource.make(IO.sleep(50.millis))(_ => IO.sleep(100.millis))
           tracer
-            .resourceSpan("Start up")(resource)
-            .surround(
-              userIdAlg
-                .getAllUsersForInstitution(
-                  "9902181e-1d8d-4e00-913d-51532b493f1b"
-                )
-                .flatMap(IO.println)
-            )
+            .spanResource("Start up")
+            .use { case (_, nt /* natural transformation */ ) =>
```

Comment:
If I'm new to otel4s, I don't understand what `nt` does, even if I know natural transformations.  Maybe we should call it `wrap`?

Reply:
I'm okay with calling it `wrap`, though I think even that is confusing. I think the example will need a comment describing its meaning and usage better regardless